app/src/pages/prompt/PromptVersionTagsList.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { graphql, useFragment } from "react-relay";
+
+import { Tag, TagGroup, TagList } from "@phoenix/components";
+
+import { PromptVersionTagsList_data$key } from "./__generated__/PromptVersionTagsList_data.graphql";
+
+export function PromptVersionTagsList({
+  promptVersion,
+}: {
+  promptVersion: PromptVersionTagsList_data$key;
+}) {
+  const data = useFragment<PromptVersionTagsList_data$key>(
+    graphql`
+      fragment PromptVersionTagsList_data on PromptVersion {
+        tags {
+          id
+          name
+        }
+      }
+    `,
+    promptVersion
+  );
+
+  const tags = data.tags.map((tag) => ({
+    id: tag.id,
+    name: tag.name,
+  }));
+  return (
+    <TagGroup aria-label="Prompt Version Tags">
+      <TagList items={tags}>{(tag) => <Tag>{tag.name}</Tag>}</TagList>
+    </TagGroup>
```

Comment:
This map seems extraneous

Reply:
yeah it seems that way but in reality the aria-components builds a collection under the hood. Without this it's not wrapped in a collection and it doesn't let you map over tags inside the list. It's weird but how it works.